src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -25,28 +28,47 @@ class GaussianIVPFilter(ODESolver):
 
     Parameters
     ----------
-    ivp
+    ivp :
         Initial value problem to be solved.
-    prior
-        Prior distribution.
-    measurement_model
-        ODE measurement model.
-    with_smoothing
+    dynamics_model :
+        dynamics_model distribution.
+    measurement_model :
+        Linearized ODE measurement model. Must be a `DiscreteEKFComponent`.
+    with_smoothing :
         To smooth after the solve or not to smooth after the solve.
-    init_implementation
+    init_implementation :
         Initialization algorithm. Either via Scipy (``initialize_odefilter_with_rk``) or via Taylor-mode AD (``initialize_odefilter_with_taylormode``).
         For more convenient construction, consider :func:`GaussianIVPFilter.construct_with_rk_init` and :func:`GaussianIVPFilter.construct_with_taylormode_init`.
-    initrv
+    initrv :
         Initial random variable to be used by the filter. Optional. Default is `None`, which amounts to choosing a standard-normal initial RV.
         As part of `GaussianIVPFilter.initialise()` (called in `GaussianIVPFilter.solve()`), this variable is improved upon with the help of the
         initialisation algorithm. The influence of this choice on the posterior may vary depending on the initialization strategy, but is almost always weak.
+    diffusion_model :
+        Diffusion model. This determines which kind of calibration is used. We refer to Bosch et al. (2020) [1]_ for a survey.
+    _repeat_after_calibration :
+        Repeat the step after the initial calibration. Improves the result of each step, but slightly increases computational complexity.
+        Optional. A default value is inferred from the type of diffusion_model model:
+        if it is a `ConstantDiffusion`, this parameter is set to `False`;
+        if it is a `PiecewiseConstantDiffusion`, this parameter is set to `True`.
```

Comment:
I'm also wondering: Do you see an argument for using the `PiecewiseConstantDiffusion`, but _not_ predicting again with the correct diffusion? If you do everything in the correct order (predict mean, measure, predict cov, compute measurement cov) then it should also not have any increased computational cost.